renderdoc/driver/gl/gl_common.h

```diff
@@ -1418,7 +1418,9 @@ enum class GLChunk : uint32_t
   glMapBufferARB,
   glMapBufferOES,
   glMapBufferRange,
+  glMapBufferRangeEXT,
   glFlushMappedBufferRange,
+  glFlushMappedBufferRangeEXT,
```

Comment:
Adding chunks in the middle of the enum like this will break backwards compatibility with all previous captures. New chunks must be added at the end of the enum. I'm not sure how these functions didn't get added before, I thought I'd added every possible GL entry point to at least reserve them, but it might be because these come from a GLES-only extension.

Reply:
I'd move the chunkds to the end of enum.